ipn/prefs.go

```diff
@@ -246,6 +246,13 @@ type Prefs struct {
 	// by name.
 	DriveShares []*drive.Share
 
+	// RelayServerPort is the UDP port number to act as a relay server on between
+	// Tailscale nodes. A nil value signifies relay server functionality
```

Comment:
what does ptr.To(0) mean?

Reply:
A random unused port will be used. Updated in https://github.com/tailscale/tailscale/pull/15594/commits/c3d6be9fa7c6b01f26ba6af456e71187f50ad02e.